groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyTextDocumentService.java

```diff
@@ -185,4 +190,63 @@ private void publishDiagnostics(Set<Diagnostic> diagnostics) {
         publishDiagnostics.accept(paramsBuilder.build());
     }
 
+    private static CompletionList createCompletionListFromSymbols(Set<SymbolInformation> symbols) {
+        CompletionListBuilder builder = new CompletionListBuilder();
+        builder.incomplete(false);
+        if (symbols == null) {
+            return builder.build();
```

Comment:
looks like the `items` field will be undefined if we don't set anything in the builder.
You can consider doing something like:

        if (symbols == null) {
            return new CompletionListImpl(false, Lists.newArrayList());
        }
        CompletionListBuilder builder = new CompletionListBuilder();
        builder.incomplete(false);
        symbols.forEach(symbol -> {
            builder.item(new CompletionItemBuilder()
                    .label(symbol.getName())
                    .kind(symbolKindToCompletionItemKind(symbol.getKind()))
                    .build());
        });
        return builder.build();